test/views/LoginViewSpec.scala

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright (c) 2022 The National Archives
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of
+ * this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+ * the Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+package views
+
+import play.api.data.{ Form, FormError }
+import play.api.i18n.MessagesApi
+import play.api.test.Helpers.stubMessagesApi
+import play.api.test.{ CSRFTokenHelper, FakeRequest, Helpers }
+import support.BaseViewSpec
+import support.CommonMatchers._
+import support.ExpectedValues.ExpectedSummaryErrorMessage
+import uk.gov.hmrc.govukfrontend.views.html.components.{ GovukButton, GovukErrorSummary, GovukFieldset }
+import uk.gov.nationalarchives.omega.editorial.forms.CredentialsFormProvider
+import uk.gov.nationalarchives.omega.editorial.models.Credentials
+import uk.gov.nationalarchives.omega.editorial.views.html.login
+
+class LoginViewSpec extends BaseViewSpec {
+  private val errorSummaryTitle = "There is a problem"
+  val defaultLang = play.api.i18n.Lang.defaultLang.code
+  private val messages: Map[String, Map[String, String]] = Map(
```

Comment:
Why are we asserting message keys in some places and the resolved messages in others? 

I'd suggest just asserting the keys, as the resolution is handled in the integration tests.

Reply:
Removed

Reply:
Confirmed.